resources/assets/js/stories/JobPosterBuilder/Intro.stories.tsx

```diff
@@ -1,14 +1,47 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
 import { withIntl } from "storybook-addon-intl";
 import { action } from "@storybook/addon-actions";
-import JobBuilderIntro from "../../components/JobBuilderIntro/JobBuilderIntro";
+import fakeJob from "../../fakeData/fakeJob";
+import IntroForm from "../../components/JobBuilderIntro/IntroForm";
+import { Job } from "../../models/types";
 
-const stories = storiesOf("Job Poster Builder|Intro", module).addDecorator(
-  withIntl,
-);
+const stories = storiesOf("Job Poster Builder|Intro", module)
+  .addDecorator(withInfo)
+  .addDecorator(withIntl);
 
-stories.add(
-  "Body",
-  (): React.ReactElement => <JobBuilderIntro handleSubmit={action("Submit")} />,
-);
+const handleSubmit = async (): Promise<Job> => {
+  action("Submit")();
+  return fakeJob();
+};
+
+stories
+  .add(
+    "Intro Body",
+    (): React.ReactElement => (
+      <IntroForm
+        job={null}
+        handleSubmit={handleSubmit}
+        handleContinueEn={action("Continue in English")}
+        handleContinueFr={action("Continue in French")}
+      />
+    ),
+    {
```

Comment:
Same with this, I added settings to the `withInfo` decorator in the storybook config file globally for all stories